commands/moderation/filter.js

```diff
@@ -0,0 +1,113 @@
+/*  
+  List of offensive words for easy readability
+  
+    Strong Words 
+      *one of these words will trigger the filter 
+       hell          damn
+       goddamn       goddamned
+       goddamnit     damned
+       damnit        hell
+       fuck          fucked
+       fucking       fucker
+       fuckwad       buttplug
+       motherfucker  wtf
+       nigger        nigga
+       penis         vagina
+       asshole       sex
+       sexed         piss
+       pissed        sexy
+       sexual        sexuality
+       bastard       clitoris
+       bitch         boobs
+       semen         sperm
+       jizzed        jizz
+       whore         prostitute
+       fornicate     fornication
+       adultery      adulter
+       adulteress    slut
+       condom        shit
+       
+    Minor Words    
+       *3+ of these words will trigger the filter
+       darn        dern
+       darned      derned
+       darnit      dernit
+       dang        heck
+       frick       fricked
+       crap        omg
+      
+*/
+
+const Discord = require('discord.js');
+
+module.exports = {
+  name: "filter",
+  description: "filter a message",
+
+  execute(msg) {
+    // Make sure user should be filtered
+    if (
+    !msg.member.roles.cache.some(
+      (role) =>
+        role.name === 'Super User' ||
+        role.name === 'Moderator' ||
+        role.name === 'Admin'
+    )
+  ) {
+      // list of words in users message
+      const words = msg.content.toLowerCase().split(' ');
+
+      // regex expression of strong offensive words
+      const strongWords = /hell|damn|goddamn|goddamned|damned|damnit|hell|fuck|fucked|fucking|fucker|fuckwad|fuckwad|motherfucker|wtf|nigger|nigga|penis|vagina|asshole|sex|sexed|piss|pissed|sexy|sexual|sexuality|bastard|bitch|boobs|semen|sperm|jizz|jizzed|whore|prostitute|fornicate|fornication|adultery|adulter|adulteress|slut|buttplug|clitoris|condom|shit[.,/\\:;\*\?\!\@\#\$]?/;
```

Comment:
Since we may change how we get all these words in the future, I won't ask you to make this change yet, but I thought it would be worth noting that this RegExp could be shortened significantly by not using the full word for each variation.
  
For example, this would cover 12 of the included words, plus some variations that use two "m"s instead of "mn":
```javascript
/((god)?dam[mn](ed|it)?)|((mother)?fuck(ed|er|ing|wad)?)/
```

Reply:
I figured it could be done a lot shorter but since we weren't certain on the word list yet, I let it long so it would be easier for me to modify. Noted for the future though.